app/src/components/map/Timebar.jsx

```diff
@@ -76,17 +75,30 @@ class Timeline extends Component {
 
   build() {
     const dummyData = this.getDummyData();
-
+    const computedStyles = window.getComputedStyle(document.getElementById('timeline_svg_container'));
```

Comment:
Ha didn't know that, thanks :clap: I'll need margins as well though so I think I'll keep  `getComputedStyle` eventually